src/app/CommandHandler.h

```diff
@@ -672,12 +690,19 @@ class CommandHandler
 
     size_t MaxPathsPerInvoke() const { return mMaxPathsPerInvoke; }
 
+    void AddToHandleList(Handle * handle);
+
+    void RemoveFromHandleList(Handle * handle);
+
+    void InvalidateHandles();
+
     bool TestOnlyIsInIdleState() const { return mState == State::Idle; }
 
     Callback * mpCallback = nullptr;
     InvokeResponseMessage::Builder mInvokeResponseBuilder;
     TLV::TLVType mDataElementContainerType = TLV::kTLVType_NotSpecified;
     size_t mPendingWork                    = 0;
+    Handle * mpHandleList                  = nullptr;
```

Comment:
we could also use IntrusiveList to handle these bits (either the chip version or pw::IntrusiveList for slightly less memory usage)

Reply:
Used IntrusiveList for this list.